src/main/java/Greet.java

```diff
@@ -0,0 +1,87 @@
+import java.util.Scanner;
+
+
+
+public class Greet {
+
+    private static Task[] list= new Task[100];
+    private static int listCounter= 0;
+    private static boolean isAwake=true;
+
+    public static void main(String[] args) {
+        System.out.println("Salutations! my name is Duke\nHow may I help you today?\n");
+        while (isAwake) {
+            Scanner in = new Scanner(System.in);
+            String input = in.nextLine();
+            String[] command = input.split(" ");
+            switch (command[0]) {
+            case "bye": {
+                System.out.println("Goodbye");
+                isAwake=false;
+                break;
+            }
+            case "list": {
+                System.out.println("Here is the list of tasks");
+                for (int i = 0; i < listCounter; i++) {
+                    System.out.println( i+1 + ". " + list[i].descTask());
+                }
+                System.out.println("there are "+ (listCounter) +" tasks in the list");
+                break;
+            }
+            case "done":{
+                if (command.length<2){
+                    System.out.println("index required");
+                    break;
+                }
+                int value=Integer.parseInt(command[1]);
+                list[value-1].setDone(true);
+                System.out.println("Nice! I've marked this task as done: "+ list[value-1].descTask());
+                break;
+            }
+            default: {
+                includeTask(input);
+            }
+            }
+        }
+    }
```

Comment:
this function is a bit long, perhaps extracts some lines for each case as a separate function